docs/_components/spinner.md

```diff
@@ -26,25 +26,3 @@ Spinners can be helpful to signal to a user that the current process may require
 </div>
 {% endcapture %}
 {% include helpers/code-example.html code=example %}
-
-## Using a button click to launch a spinner
-
-We may want to start with a hidden spinner and then remove it with a button click will the page transition resolves. 
-
-{% capture example %}
-<button class="usa-button__lg-invokeSpinner">Hide Button and trigger a spinner</button>
-<div class="lg-spinner lg-spinner--hidden" id="spinner-id">
-  <div>
-    <img
-      src="{{ site.baseurl }}/assets/img/spinner.gif"
```

Comment:
should we remove the `spinner.gif` and `@2x` files from this repo as well?

Reply:
> should we remove the `spinner.gif` and `@2x` files from this repo as well?

The spinner component itself will continue to exist after this, just not the button variant. We're still using the spinner in at least one place in the IdP (IAL2 Acuant modal overlay loading state).